UnityMirror/UnityServer/Assets/PlayFabSdk/Shared/Internal/PlayFabErrors.cs

```diff
@@ -453,25 +453,61 @@ public enum PlayFabErrorCode
         EntityAPIKeyOrSecretInvalid = 1449,
         EconomyServiceUnavailable = 1450,
         EconomyServiceInternalError = 1451,
-        KustoProxyQueryRateLimitExceeded = 1455,
-        EntityAPIKeyCreationDisabledForEntity = 1456,
-        StudioCreationRateLimited = 1457,
-        StudioCreationInProgress = 1458,
-        DuplicateStudioName = 1459,
-        StudioNotFound = 1460,
-        StudioDeletionInProgress = 1461,
-        StudioDeactivated = 1462,
+        QueryRateLimitExceeded = 1452,
+        EntityAPIKeyCreationDisabledForEntity = 1453,
+        ForbiddenByEntityPolicy = 1454,
+        UpdateInventoryRateLimitExceeded = 1455,
+        StudioCreationRateLimited = 1456,
+        StudioCreationInProgress = 1457,
+        DuplicateStudioName = 1458,
+        StudioNotFound = 1459,
+        StudioDeleted = 1460,
+        StudioDeactivated = 1461,
+        StudioActivated = 1462,
```

Comment:
Are these just copied from pf-main?

Reply:
Or this is generated?

---
In reply to: [408517602](https://github.com/PlayFab/gsdkSamples/pull/13#discussion_r408517602) [](ancestors = 408517602)

Reply:
they are part of the PlayFab Unity SDK, which is used in this sample.